.travis.yml

```diff
@@ -3,7 +3,7 @@ python:
   - "3.6"
   - "3.7"
 before_install:
-  - "pip install -U pip setuptools virtualenv python-coveralls fmf click mock"
+  - "pip install -U pip setuptools virtualenv python-coveralls fmf click mock nitrate"
   - "sudo apt-get -y install vagrant"
```

Comment:
So this does not work/help because of the missing config. Should be removed.

Reply:
Reverted by 31646f8